### ISSUE TITLE
Enhance suricata settings

### ISSUE THREAD

**Issue by CallMeR** [opnsense/core#5143]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

- [*] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [* ] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

When useing suricata in opnsense , the log shows 

> suricata[94223] | [100139] <Notice> -- all 2 packet processing threads, 4 management threads initialized, engine started.

But in suricata's official web site：

<img width="754" alt="image" src="https://user-images.githubusercontent.com/11908641/128458873-5654d20a-f3eb-424a-bee4-8aca245d0152.png">
<img width="720" alt="image" src="https://user-images.githubusercontent.com/11908641/128460012-e402dece-5ce7-459a-a34e-322d378238fb.png">

Although I am not sure whether numbers of "packet processing threads" will affect performance, I hope to enhance the ability to adjust this parameter  :)

**Comment by AdSchellevis** [comment]
I don't expect it will trigger a substantial difference, but you can easily test adding the setting in `/usr/local/etc/suricata/custom.yaml`

**Comment by CallMeR** [comment]
Thanks for your reply :)

When I am running some benchmark，it seem like suricata only use 1/4 cpu cores: 
<img width="746" alt="image" src="https://user-images.githubusercontent.com/11908641/128475694-9765dbbd-6540-4ae8-9b52-c22d08f39d6a.png">

my cpu is:
<img width="594" alt="image" src="https://user-images.githubusercontent.com/11908641/128475858-d65900c2-4f55-4267-91a1-3b5acbf1d4df.png">

I really want to know that if Suricata can use all my CPU cores

**Comment by AdSchellevis** [comment]
in IPS mode that's currently logica, since netmap communicates with a single thread to the host stack. They are working on using the new api, which should improve the situation (https://redmine.openinfosecfoundation.org/issues/4541)

**Comment by QY-Ryan-Chen** [comment]
> in IPS mode that's currently logica, since netmap communicates with a single thread to the host stack. They are working on using the new api, which should improve the situation (https://redmine.openinfosecfoundation.org/issues/4541)

https://github.com/luigirizzo/netmap/commit/d02e4563570c2785cb4d67ce7281f020b475f345

These API about libnetmap.h ？

**Comment by CallMeR** [comment]
I recently upgraded opnsense to version 21.7.3-amd64. The version of Suricata is 6.0.3 .
And I did several bandwidth tests with Speedtest and iperf3.
CPU：J4125
Mem：4GB
Virtualization：ESXi 7.0 latest
nic：vmxnet3

<img width="691" alt="Snipaste_2021-09-24_21-51-44" src="https://user-images.githubusercontent.com/11908641/134686848-4788e74a-a7e9-461f-8506-c084c604f745.png">

Suricata's matching pattern is "hyperscan", when WAN is selected as the interface, I can get full speed bandwidth（over 500Mbps）, when LAN is selected as the interface, I can get 250~270Mbps bandwidth. CPU utilization is shown in the figure above.


The upstream shows that the problem of netmap has been solved and the code has been merged
https://github.com/OISF/suricata/pull/6372

Is there a problem with my settings on Suricata？
